Ginger/GingerCoreCommon/VariablesLib/VariablePasswordString.cs

```diff
@@ -64,7 +64,10 @@ public override void GenerateAutoValue()
 
         public override List<VariableBase.eSetValueOptions> GetSupportedOperations()
         {
-            throw new System.NotImplementedException();
+            List<VariableBase.eSetValueOptions> supportedOperations = new List<VariableBase.eSetValueOptions>();
```

Comment:
why this was added?
Password variable value can't be set during run time or been reset. user need to set it offline as part of flow design.

Reply:
Earlier it was not there and it is pulling all the string operations to this one which password is not supporting. When I was writing TC's it is showing ClearSpecialChar as one of the supported operation and PasswordString doesn't have that implementation. So I added this validation to ensure that it shows only the supported operations.

Reply:
Those operation are shown as possible operation in SetVariableValue action, right? so after this addition those "SetValue" and "ResetValue" will appear as possible operations in the Action and they not supposed to be shown.
let me know what i'm missing

Reply:
Reverted back to throw NotImplementedException exception